aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModuleAgent.java

```diff
@@ -2609,18 +2609,12 @@ long quorumPosition()
     int updateLeaderPosition(final long nowNs, final long position)
     {
         thisMember.logPosition(position).timeOfLastAppendPositionNs(nowNs);
-        final long commitPosition = min(ClusterMember.quorumPosition(activeMembers, rankedPositions), position);
+        final long commitPosition = min(quorumPosition(), position);
 
         if (commitPosition > this.commitPosition.getWeak() ||
             nowNs >= (timeOfLastLogUpdateNs + leaderHeartbeatIntervalNs))
         {
-            for (final ClusterMember member : activeMembers)
-            {
-                if (member.id() != memberId)
-                {
-                    consensusPublisher.commitPosition(member.publication(), leadershipTermId, commitPosition, memberId);
-                }
-            }
+            sendCommitPosition(commitPosition);
```

Comment:
`sendCommitPosition -> publishCommitPosition` to be consistent with existing naming.